plugins/wordpress-rule-exclusions-before.conf

```diff
@@ -634,6 +637,17 @@ SecRule REQUEST_FILENAME "@rx /wp-admin/(?:admin|admin-ajax|edit|users)\.php$" \
     ctl:ruleRemoveTargetById=932236;ARGS_NAMES:ids,\
     ver:'wordpress-rule-exclusions-plugin/1.0.1'"
 
+# Managing users
+SecRule REQUEST_FILENAME "@endsWith /wp-admin/users.php" \
+    "id:9507602,\
+    phase:1,\
+    pass,\
+    t:none,\
+    nolog,\
+    ver:'wordpress-rule-exclusions-plugin/1.0.1',\
+    ctl:ruleRemoveTargetById=920273;ARGS_NAMES:users[0],\
+    ctl:ruleRemoveTargetById=942432;ARGS_NAMES:users[0]"
+
```

Comment:
@EsadCetiner What about rule `9507601`?

Reply:
the false positive only happens on this endpoint, if we're going to move it to `9507601` then we'd might as well move it back to where it was.

Reply:
It's not the same. Before, you were whitelisting it for the whole site (for any URL as rule `9507350` is using `@unconditionalMatch`, have you noticed that?). Moving it to rule `9507601` will white list it only for 4 endpoints.

Reply:
But those other 3 endpoints don't make use of it. I put this rule exclusions in `9507350` originally since it's pointless to create a new rule for it (There is nothing to gain from it outside of complexity and a performance hit), and if I were to add it to `9507601` I'd be allowing it on more endpoints then it needs to be allowed on. The rule exclusion is just disabling the rules for `ARGS_NAMES:users[0]` and not `ARGS:users[0]` so I'm not sure why we have to be more granular here, as soon as the `ARGS_NAMES` changes the rule won't be applied, and it's not like `users[0]` is an SQL injection attack or something, it's harmless.

Reply:
Why you choosed `9507350` and not `9507601`?

Reply:
@azurit Both rules are allowing `ARGS_NAMES:users[0]` more than it needs to be allowed. `9507601` will be deleted at some point when the false positives with `932236` (There's an open PR for this) is fixed anyways, and then that one `ARGS_NAMES:users[0]` rule exclusion will stay there with an unnecessarily complex regex?